### ISSUE TITLE
Datasets URL error

### ISSUE THREAD

**Issue by Patrick00017** [monniert/unicorn#7]
CUB and LSUN dataset URL are broken. Help!

**Comment by monniert** [comment]
Indeed, here is the new one for CUB (https://data.caltech.edu/records/65de6-vp158/files/CUB_200_2011.tgz?download=1), for LSUN it seems the official website is broken, so there is nothing I can do... I will try to check it later
